package-lock.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Why there is package-lock.json in root project? isn't it should be in either backend or learn-arai?